### DISCUSSION TITLE
Is it possible to disable the --graph by default in the log view?

### DISCUSSION THREAD

**Discussion by synic** [NeogitOrg/neogit#1197] [Q&A] [answered]
Something I missed from magit, if you've got a lot of developers and merging the graph view can be very messy, so I prefer to disable it. Is there a way to do it by default so I don't have to do it every time I view the log?

**Comment by CKolkey** [comment, marked as answer]
If you have "remember settings" enabled it should persist. Is that not working?

**Reply by synic** [reply]
Wow, not sure how I missed that. Now I'm wondering if that's how I did it in Magit too? Thanks!

**Reply by CKolkey** [reply]
Not sure how magit handles that, but glad to hear it works as intended :)
